dragula.js

```diff
@@ -58,8 +59,13 @@ function dragula (initialContainers, options) {
 
   return drake;
 
-  function isContainer (el) {
-    return drake.containers.indexOf(el) !== -1 || o.isContainer(el);
+  function isContainer (el, handle) {
+    handle = handle || null;
+    if (handle && o.allowNestedContainers) {
+       return drake.containers.indexOf(el) !== -1 && o.isContainer(el, handle);
```

Comment:
You reckon? Havent worked on this in quite a while, so I can give it a look over again, thanks for the review!